en-AU/auth.json

```diff
@@ -105,6 +107,8 @@
       "disclaimer_privacy_link": "Privacy Policy",
       "disclaimer_terms_of_use_link": "Terms of use",
       "disclaimer_clickwrap_input_label": "Accept policies",
+      "disclaimer_clickwrap_heading": "Almost there",
+      "disclaimer_clickwrap_description": "To continue, we need you to read and accept our policies.",
```

Comment:
@viv-kinde, the convention is not to use a full stop for standalone sentences. Despite being some outliers, most of our English page descriptions adhere to this. Thx